it/mail/chrome/messenger/preferences/sendoptions.dtd

```diff
@@ -24,16 +24,16 @@
 <!ENTITY DeleteHtmlDomain.accesskey   "l">
 <!ENTITY DeletePlainText.accesskey    "t">
 
-<!ENTITY add.htmltitle                "Aggiungere un dominio HTML:">
+<!ENTITY add.htmltitle                "Aggiungi un nome dominio HTML:">
 <!ENTITY add.htmldomain               "Nome dominio HTML:">
 <!ENTITY add.htmldomain.accesskey     "d">
 <!ENTITY add.plaintexttitle           "Aggiungi un nome dominio testo semplice:">
 <!ENTITY add.plaintextdomain          "Nome dominio testo semplice:">
 <!ENTITY add.plaintext.accesskey      "d">
 
 <!-- LOCALIZATION NOTE: do not translate @string@ -->
-<!ENTITY domainnameError.title        "Errore nel nome di dominio">
-<!ENTITY duplicationError.label       "The domain name @string@ already exists in this list.">
+<!ENTITY domainnameError.title        "Errore nome dominio">
+<!ENTITY duplicationError.label       "Il nome dominio @string@ esiste già in questa lista.">
 
 <!-- LOCALIZATION NOTE: do not translate @string@ -->
-<!ENTITY dualEntryError.label         "A domain name cannot appear in both the HTML and Plain Text lists. The domain name @string@ already exists.">
+<!ENTITY dualEntryError.label         "Un nome dominio non può apparire sia in liste HTML sia in liste Testo semplice. Il nome dominio @string@ esiste già.">
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+<!ENTITY dualEntryError.label         "Un nome dominio non può essere presente sia in liste HTML sia in liste Testo semplice. Il nome dominio @string@ esiste già.">